golem-worker-executor-base/src/services/scheduler.rs

```diff
@@ -71,6 +79,24 @@ impl<Ctx: WorkerCtx> SchedulerWorkerAccess for Arc<dyn WorkerActivator<Ctx> + Se
             .await?;
         Ok(worker.oplog())
     }
+
+    async fn activate_and_invoke(
+        &self,
+        owned_worker_id: OwnedWorkerId,
+        idempotency_key: IdempotencyKey,
+        full_function_name: String,
+        function_input: Vec<Value>,
+    ) -> Result<(), GolemError> {
+        let worker = self
+            .get_or_create_running(&owned_worker_id, None, None, None, None)
+            .await?;
+
+        worker
+            .invoke(idempotency_key, full_function_name, function_input)
+            .await?;
+
```

Comment:
Given the name is`activate_and_invoke`, what is getting "activated" here?

Reply:
@afsalthaj it activates the worker to be able to invoke it. this terminology is used in many places in the executor, no need to change here.